src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
```

Comment:
Maybe it will be better to put inbuilt java classes on top of custom packages to follow the coding standards